molvis/vasp/python/processors/chgcarsource.py

```diff
@@ -70,6 +70,14 @@ def __init__(self, id, name):
         self.properties.customDataRange.semantics = ivw.properties.PropertySemantics(
             "Text")
 
+        self.changeSign = ivw.properties.BoolProperty(
+            "changeSign", "Change the sign of the charge", False)
+        self.addProperty(self.changeSign)
+
+        self.offsetData = ivw.properties.BoolProperty(
+            "Offset", "Offset the data", True)
```

Comment:
A bit hard to understand what kind of offset that will be applied. I think "Center" would be more telling?

Probably should not use offset for the name as it imples that it is an offset. Also, should start with lower case.
```suggestion
            "centerObject", "Center XXX", True)
```